kotlin_money/src/main/java/com/github/paylike/kotlin_money/Money.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.paylike.kotlin_money
+
+import com.github.paylike.kotlin_currencies.PaylikeCurrency
+
+object Money {
+    private const val maxInt: Long = Long.MAX_VALUE // 9223372036854775807L
```

Comment:
Thi should be `9007199254740991`, we won't be able to process anything above that.

Reply:
Done